packages/terra-table/src/subcomponents/_HeaderCheckMarkCell.jsx

```diff
@@ -112,14 +112,14 @@ const HeaderCheckMarkCell = ({
         onKeyDown: wrappedOnKeyDownForItem(onKeyDown, onSelect, metaData),
         onBlur: wrappedEventCallback(onBlur, event => event.currentTarget.setAttribute('data-cell-show-focus', 'true')),
         onMouseDown: wrappedEventCallback(onMouseDown, event => event.currentTarget.setAttribute('data-cell-show-focus', 'false')),
-        tabIndex: '0',
         'data-cell-show-focus': true,
       };
     }
 
     attrCheck = {
       role: 'checkbox',
       'aria-checked': isSelected && isIndeterminate ? 'mixed' : isSelected,
+      tabIndex: '0',
```

Comment:
Should this index account for isDisabled with a -1 index?

Reply:
updated here https://github.com/cerner/terra-core/pull/3126/commits/bc141da851635bf0c558673882deb0dfafdac8ca